akka-docs/src/main/paradox/java/io-tcp.md

```diff
@@ -1,326 +0,0 @@
-# Using TCP
-
-The code snippets through-out this section assume the following imports:
-
-@@snip [IODocTest.java]($code$/java/jdocs/io/japi/IODocTest.java) { #imports }
-
-All of the Akka I/O APIs are accessed through manager objects. When using an I/O API, the first step is to acquire a
-reference to the appropriate manager. The code below shows how to acquire a reference to the `Tcp` manager.
-
-@@snip [EchoManager.java]($code$/java/jdocs/io/japi/EchoManager.java) { #manager }
```

Comment:
Is the question where this is in the combined .md file?

```
-@@snip [EchoManager.java]($code$/java/jdocs/io/japi/EchoManager.java) { #manager }
```

If so, it is found in the [diff from the 1st git commit](https://github.com/richard-imaoka/akka/commit/de6e2007a40e0db1935dce7ac8b38fc268567c08#diff-5c2f49192b560945210a9e5b9a079439).

However, 

![image](https://user-images.githubusercontent.com/7414320/27410364-004ab07a-5722-11e7-99f1-e9d26bbd278a.png)

somehow "Files Changed" of the entire pull request doesn't show the diff from scala/io-tcp.md. 

This has happened sometimes when changing java/***.md to a symlink pointing to scala/***.md. Even if I squash two commits into one, this still happens. I don't know why...